ansible_community_steering_committee.rst

```diff
@@ -70,6 +70,38 @@ Depending on a topic you want to discuss with the Community and the Committee, a
 * `Ansible Collection Requirements <https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst>`_.
 * `Ansible Collection Inclusion Checklist <https://github.com/ansible-collections/overview/blob/main/collection_checklist.md>`_.
 
+Community Topics Workflow
+-------------------------
+
+The Committee uses the `Community-topics workflow <https://github.com/ansible-community/community-topics/blob/main/community_topics_workflow.md>`_ to asynchronously discuss and vote on the `community-topics <https://github.com/ansible-community/community-topics/issues>`_.
+
+The quorum, the minimum number of Committee members who must vote on a topic in order for a decision to be officially made, is half of the whole number of the Committee members. 
+
+If the quorum number contains a fractional part, it is rounded using standard rules for rounding whole numbers. For example, if there are thirteen members currently in the committee, the quorum will be seven.
```

Comment:
```suggestion
If the quorum number contains a fractional part, it is rounded up to the next whole number. For example, if there are thirteen members currently in the committee, the quorum will be seven.
```
There are two 'standard rules' for rounding to whole numbers: rounding up once the fractional part is >= 0.5, and round half to even where 0.5 is rounded up or down depending on whether the next integer is even or odd (https://en.wikipedia.org/wiki/Rounding#Round_half_to_even).

Let's define something explicit here which is simple, like always rounding up :)

Reply:
I like it, sounds fair:)